qsr_lib/src/qsrlib_qsrs/qsr_qtc_bc_simplified.py

```diff
@@ -54,9 +54,21 @@ def make(self, *args, **kwargs):
         else:
             qsrs_for = self._return_all_possible_combinations(sorted(input_data.trace[timestamps[0]].objects.keys()))
 
+        combinations = {}
         if qsrs_for:
             for p in qsrs_for:
                 between = str(p[0]) + "," + str(p[1])
```

Comment:
@pet1330 and I did some test on the timing and ran the following tests 1,000,000 times. The time is the sum of all theses runs:

`str(p[0]) + "," + str(p[1])`: 0.0990860462189
`",".join(p)`: 0.0934491157532
`",".join(reversed(p))`: 0.405081033707

So there is virtually no difference between `+` and `join()` but the `reversed` makes it significantly slower.